let p = new Promise ((resolve, reject) => {
//codigo pendente
let x = 5;
let y = 6;
let z = x + y;

if (z == 10) {
  resolve ('ok');
} else {
  reject('estÃ¡ errado');
}
});

console.log('start');

p.then(val =>{ console.log(val)})
 .catch( err => console.log(err))